tests/mmptest/src/MMPTest.cs

```diff
@@ -774,5 +774,31 @@ public void OldXcodeTest ()
 				TI.TestUnifiedExecutable (test, shouldFail: false, configuration: "Debug", environment: new string [] { "MD_APPLE_SDK_ROOT", Path.GetDirectoryName (Path.GetDirectoryName (oldXcode)) });
 			});
 		}
+
+		[Test]
+		public void BuildUnified_BelowMinDeploymentTargetShouldFail ()
+		{
+			RunMMPTest (tmpDir => {
+				TI.UnifiedTestConfig test = new TI.UnifiedTestConfig (tmpDir) {
+					PlistReplaceStrings = new Dictionary<string, string> {
+						{ "<string>10.7</string>", "<string>10.4</string>"}
+					}
+				};
+				TI.TestUnifiedExecutable (test, shouldFail: true);
+			});
+		}
+
+		[Test]
+		public void BuildUnified_AboveSDKDeploymentTargetShouldFail ()
+		{
+			RunMMPTest (tmpDir => {
+				TI.UnifiedTestConfig test = new TI.UnifiedTestConfig (tmpDir) {
+					PlistReplaceStrings = new Dictionary<string, string> {
+						{ "<string>10.7</string>", "<string>11.0</string>"}
+					}
+				};
+				TI.TestUnifiedExecutable (test, shouldFail: true);
+			});
+		}
```

Comment:
Please write a non-failing test where you set it to 10.11 or something and assert it works. Only failing tests can still pass when you have an actual error.